exp/api/v1beta1/gcpmanagedcluster_types.go

```diff
@@ -59,6 +59,10 @@ type GCPManagedClusterSpec struct {
 	// supplied then the credentials of the controller will be used.
 	// +optional
 	CredentialsRef *infrav1.ObjectReference `json:"credentialsRef,omitempty"`
+
+	// LoadBalancerSpec contains configuration for one or more LoadBalancers.
+	// +optional
+	LoadBalancer infrav1.LoadBalancerSpec `json:"loadBalancer,omitempty"`
```

Comment:
Typo `LoadBalancerSpec` on the godoc vs `LoadBalancer` on the actual field.

Reply:
Thanks for catching that. Fixed.